codex-process-data-transfer/src/main/java/de/netzwerk_universitaetsmedizin/codex/processes/data_transfer/service/send/ResolvePsn.java

```diff
@@ -123,6 +148,15 @@ private String resolveBloomFilter(String bloomFilter)
 						"Unable to get DIC pseudonym for given BloomFilter"));
 	}
 
+	private String resolveLocalPseudonym(String localPseudonym)
+	{
+		return fttpClientFactory.getFttpClient().getDicPseudonymForLocalPseudonym(localPseudonym)
+				.orElseThrow(() -> new BpmnError(
+						CODESYSTEM_NUM_CODEX_DATA_TRANSFER_ERROR_VALUE_NO_DIC_PSEUDONYM_FOR_BLOOMFILTER,
```

Comment:
This error should have a specific error code. Add a new code to the http://www.netzwerk-universitaetsmedizin.de/fhir/CodeSystem/data-transfer-error CodeSystem.